src/pyocr/builders.py

```diff
@@ -329,7 +331,7 @@ def write_file(file_descriptor, text):
     def start_line(self, box):
         self.built_text.append(u"")
 
-    def add_word(self, word, box):
+    def add_word(self, word, box, confidence=None):
```

Comment:
1) Either make `confidence` optional for all `add_word()` or for none. But the API of the builders must be the same for all of them.
2) I would recommend '0' as default value instead. Seems safer.

Reply:
I think you missed my point 1 :-)
Either you specify a default value for `confidence` on *all* the Builders.add_word() methods (making the argument optional), or on none of them. They must *all* have the same API (see the base class `builders.BaseBuilder`)

Reply:
Ah sorry I did miss you point! The parameter has now been made optional for all `add_word` methods